src/library/Uno.Material/Styles/Controls/v2/PasswordBox.xaml

```diff
@@ -362,11 +363,11 @@
 
 								<VisualState x:Name="Focused">
 									<VisualState.Setters>
-										<Setter Target="Root.BorderBrush"
+										<Setter Target="RootBorder.BorderBrush"
 												Value="{StaticResource PrimaryBrush}" />
 										<Setter Target="PlaceholderElement.Foreground"
 												Value="{StaticResource PrimaryBrush}" />
-										<Setter Target="Root.BorderThickness"
+										<Setter Target="RootBorder.BorderThickness"
```

Comment:
everywhere where you are changing this BorderThickness, you need to adjust the padding accordingly